From 0d21e16aa3d5c4bc292173c53c0609a31a0d6b91 Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Mon, 12 May 2014 23:45:53 -0700
Subject: [PATCH 4061/5965] Remove unused GPU driver directory

MALI GPU driver is distributed as a standalone driver package. Remove unused
file from compilation leftover.
---
 drivers/amlogic/mali/__malidrv_build_info.c   |  1 -
 .../amlogic/mali/arch/arch-aml-meson-m400-1   |  1 -
 .../amlogic/mali/arch/arch-aml-meson-m400-2   |  1 -
 drivers/amlogic/mali/arch/config.h            | 75 -------------------
 4 files changed, 78 deletions(-)
 delete mode 100644 drivers/amlogic/mali/__malidrv_build_info.c
 delete mode 120000 drivers/amlogic/mali/arch/arch-aml-meson-m400-1
 delete mode 120000 drivers/amlogic/mali/arch/arch-aml-meson-m400-2
 delete mode 100755 drivers/amlogic/mali/arch/config.h

diff --git a/drivers/amlogic/mali/__malidrv_build_info.c b/drivers/amlogic/mali/__malidrv_build_info.c
deleted file mode 100644
index d5ac68f35232..000000000000
--- a/drivers/amlogic/mali/__malidrv_build_info.c
+++ /dev/null
@@ -1 +0,0 @@
-const char *__malidrv_build_info(void) { return "malidrv:  CONFIG=aml-meson-m400-2 USING_OS_MEMORY=1 API_VERSION=17 REPO_URL= REVISION= CHANGED_REVISION= CHANGE_DATE= BUILD_DATE=Wed Nov 28 19:14:47 CST 2012 BUILD=RELEASE CPU= USING_UMP=1 USING_MALI200=0 USING_MALI400=3 USING_MALI400_L2_CACHE=1 USING_GP2=0 KDIR= MALI_PLATFORM_FILE=platform/meson_platform/mali_platform.c platform/meson_platform/mali_fix.c OS_MEMORY_KERNEL_BUFFER_SIZE_IN_MB=16 USING_PROFILING=0 USING_INTERNAL_PROFILING=0 USING_GPU_UTILIZATION=";}
diff --git a/drivers/amlogic/mali/arch/arch-aml-meson-m400-1 b/drivers/amlogic/mali/arch/arch-aml-meson-m400-1
deleted file mode 120000
index b2d568a2fb55..000000000000
--- a/drivers/amlogic/mali/arch/arch-aml-meson-m400-1
+++ /dev/null
@@ -1 +0,0 @@
-arch-aml-meson-m400-1
\ No newline at end of file
diff --git a/drivers/amlogic/mali/arch/arch-aml-meson-m400-2 b/drivers/amlogic/mali/arch/arch-aml-meson-m400-2
deleted file mode 120000
index b474ad111201..000000000000
--- a/drivers/amlogic/mali/arch/arch-aml-meson-m400-2
+++ /dev/null
@@ -1 +0,0 @@
-arch-aml-meson-m400-2
\ No newline at end of file
diff --git a/drivers/amlogic/mali/arch/config.h b/drivers/amlogic/mali/arch/config.h
deleted file mode 100755
index f263bcd2ee46..000000000000
--- a/drivers/amlogic/mali/arch/config.h
+++ /dev/null
@@ -1,75 +0,0 @@
-/*
- * Copyright (C) 2012 Amlogic, Inc. All rights reserved.
- * 
- * This program is free software and is provided to you under the terms of the GNU General Public License version 2
- * as published by the Free Software Foundation, and any use by you of this program is subject to the terms of such GNU licence.
- * 
- * A copy of the licence is included with the program, and can also be obtained from Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-
-
-#ifndef __ARCH_CONFIG_H__
-#define __ARCH_CONFIG_H__
-
-#define INT_MALI_GP	48
-#define INT_MALI_GP_MMU 49
-#define INT_MALI_PP	50
-#define INT_MALI_PP_MMU 51
-
-/* Configuration for the AMLOGIC MESON platform */
-
-static _mali_osk_resource_t arch_configuration [] =
-{
-	{
-		.type = MALI400GP,
-		.description = "Mali-400 GP",
-		.base = 0xd0060000,
-		.irq = INT_MALI_GP,
-		.mmu_id = 1
-	},
-	{
-		.type = MALI400PP,
-		.base = 0xd0068000,
-		.irq = INT_MALI_PP,
-		.description = "Mali-400 PP",
-		.mmu_id = 2
-	},
-#if USING_MMU
-	{
-		.type = MMU,
-		.base = 0xd0063000,
-		.irq = INT_MALI_GP_MMU,
-		.description = "Mali-400 MMU for GP",
-		.mmu_id = 1
-	},
-	{
-		.type = MMU,
-		.base = 0xd0064000,
-		.irq = INT_MALI_PP_MMU,
-		.description = "Mali-400 MMU for PP",
-		.mmu_id = 2
-	},
-#endif
-	{
-		.type = OS_MEMORY,
-		.description = "Mali Memory",
-		.alloc_order = 5, /* Medium preference for this memory */
-		.size = 0x10000000,
-		.flags = _MALI_CPU_WRITEABLE | _MALI_CPU_READABLE | _MALI_PP_READABLE | _MALI_PP_WRITEABLE |_MALI_GP_READABLE | _MALI_GP_WRITEABLE
-	},
-	{
-		.type = MEM_VALIDATION,
-		.description = "Framebuffer",
-		.base = 0x84000000,
-		.size = 0x06000000,
-		.flags = _MALI_CPU_WRITEABLE | _MALI_CPU_READABLE | _MALI_PP_WRITEABLE | _MALI_PP_READABLE
-	},
-	{
-		.type = MALI400L2,
-		.base = 0xD0061000,
-		.description = "Mali-400 L2 cache"
-	},
-};
-
-#endif /* __ARCH_CONFIG_H__ */
-- 
2.19.0

